docs/zdgbook/GettingStarted.rst

```diff
@@ -2,142 +2,141 @@
+  parts =
+      zope4
+
+  [zope4]
+  recipe = zc.recipe.egg
+  eggs =
+      Zope
 
-Installing Zope 2
-=================
+The ``[zope4]`` part uses ``zc.recipe.egg`` which will download
+``Zope`` and all its dependencies.  It will create few console
+scripts inside the ``bin`` directory.
 
-Zope 2 is distributed in egg format.  To install Zope 2 egg
-and create an instance, update buildout configuration file
-(``buildout.cfg``) with appropriate parts and recipes.
+After updating the buildout configuration, you can run the `buildout`
+command to build the system.
 
 ::
 
-  [buildout]
-  parts = zope2
-          instance
-  extends = http://download.zope.org/Zope2/index/2.13.20/versions.cfg
+  $ buildout
 
-  [zope2]
-  recipe = zc.recipe.egg
-  eggs = Zope2
-  interpreter = zopepy
+The initial build will take some time to complete.
 
-  [instance]
-  recipe = plone.recipe.zope2instance
-  user = admin:admin
-  http-address = 8080
-  eggs = ${zope2:eggs}
+.. attention::
 
-The ``[zope2]`` part use `zc.recipe.egg` which will download `Zope2`
-egg and all its dependencies.  It will create few console scripts
-inside `bin` directory.  Also it will create a custom Python
-interpreter named ``zopepy``.
+  You will see some warnings or error messages on the console. These
+  cannot be fixed until Python 2 support is dropped. Just ignore
+  them for now.
 
-The ``[instance]`` part creates a Zope 2 application instance to
-develop application.  It will create a script named ``instance``
-inside `bin` directory.  We can use that script to run the
-application instance.
+Creating the instance
+=====================
 
-After updating the buildout configuration, you can run the `buildout`
-command to build the system.
+Once the build is complete, you can create an instance as follows.
 
 ::
 
-  $ ./bin/buildout
+  $ bin/mkwsgiinstance -d .
 
-The initial build will take some time to complete.
 
-Running Instance
-================
+Running the instance
+====================
 
-Once build is completed, you can run Zope 2 instance like this.
+Once you got a Zope instance, you can run it like this.
 
 ::
 
-  $ ./bin/instance fg
-
+  $ bin/runwsgi etc/zope.ini
 
-You can see that Zope is running in 8080 port.  You can go to the
-Zope Management Interface (ZMI).
+Now, Zope is running. You can convince yourself by visiting the
+following URL.
 
 ::
 
-  http://localhost:8080/manage
+  http://localhost:8080
 
-You can provide the user name & password provided in `[instance]`
-part to access this page.
+You can also visit the adminstration area.
 
-You can see a list of installable applications in the drop-down box.
-Also you can see it in "Control_Panel" -> "Products".
+Use the user name and password you set earlier.
 
 ::
 
-  http://localhost:8080/Control_Panel/Products/manage_main
+  http://localhost:8080/manage
+
+You can see a list of installable applications in the drop-down box
```

Comment:
I think this are not applications but objects which can be created in the ZODB.

Reply:
Thank you!  "applications" was the wording of the previous author, and I was not entirely sure about the naming convention. Changed it to "objects".